tests/by-util/test_date.rs

```diff
@@ -492,3 +492,101 @@ fn test_date_empty_tz() {
         .succeeds()
         .stdout_only("UTC\n");
 }
+
+#[test]
+fn test_date_tz_utc() {
```

Comment:
Can you do a blank TZ too?

Reply:
Thanks for the comments. there was one test empty TZ and I add another one with `.arg("@0")` as an additional test.